run_tests.sh

```diff
@@ -18,6 +18,7 @@ function show_help()
     '         Limit tests to unit tests' \
     '' \
     'COMMANDS' \
+    '  clear-cache - clears tests cache' \
```

Comment:
Don't forget to update documentation/content/tests.rst with this new option.

Reply:
ok